sklearn/decomposition/_kernel_pca.py

```diff
@@ -419,6 +419,7 @@ def fit(self, X, y=None):
         if self.fit_inverse_transform and self.kernel == "precomputed":
             raise ValueError("Cannot fit_inverse_transform with a precomputed kernel.")
         X = self._validate_data(X, accept_sparse="csr", copy=self.copy_X)
+        self._gamma = 1 / X.shape[1] if self.gamma is None else self.gamma
```

Comment:
I think `self.gamma_` would be a more appropriate name since it is an attribute determined at `fit`